nilearn/maskers/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,266 @@
+"""
+Transformer for computing ROI signals of multiple 4D images
+"""
+
+import itertools
+
+from joblib import Memory, Parallel, delayed
+
+from .._utils import CacheMixin
+from .._utils.niimg_conversions import _iter_check_niimg
+from .nifti_labels_masker import NiftiLabelsMasker
+
+
+class MultiNiftiLabelsMasker(NiftiLabelsMasker, CacheMixin):
+    """Class for masking of Niimg-like objects.
+    MultiNiftiLabelsMasker is useful when data from non-overlapping volumes
+    and from different subjects should be extracted (contrary to
+    NiftiLabelsMasker).
+    Parameters
+    ----------
+    labels_img: Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Region definitions, as one image of labels.
+
+    labels : list of str, optional
+        Full labels corresponding to the labels image. This is used
+        to improve reporting quality if provided.
+        Warning: The labels must be consistent with the label
+        values provided through `labels_img`.
+
+    background_label: number, optional
+        Label used in labels_img to represent background.
+
+    mask_img: Niimg-like object, optional
```

Comment:
I guess some of the fields of the docstring can be filled in automatically. See the docuemntation of NiftiLabelsMasker.